test/Mono.Linker.Tests/TestCasesRunner/ResultChecker.cs

```diff
@@ -729,10 +729,16 @@ void VerifyLoggedMessages (AssemblyDefinition original, LinkerTestLogger logger,
 										if (sourceColumn != null && mc.Origin?.SourceColumn != sourceColumn.Value)
 											return false;
 									} else {
+										if (attrProvider.FullName.Contains ("PropertyWithUnsupportedType"))
+											System.Diagnostics.Debug.WriteLine ("");
+
```

Comment:
Do we need this? Looks like it was left here accidentally.

Reply:
Story of me debugging the linker :-(